### PR TITLE
boards: arm: Updated Cypress PSoC6 cy8ckit_062_wifi_bt_m0 doc

### PR DESCRIPTION
**Pull request by OlegKapshii**
Added description how to build, programm and see console output of
hello_world example on cy8ckit_062_wifi_bt_m0 board.

Signed-off-by: Oleg Kapshii <oleg.kapshii@cypress.com>

### CODE REVIEW

**Comment by OlegKapshii** [comment]
There is some error in Shippable, but I do not understand what is wrong here and what should I do: "77144675eacbe65478536e0adc3bf03d737090f2: author email (Signed-off-by: Oleg Kapshii <oleg.kapshii@cypress.com>) needs to match one of the signed-off-by entries"

**Comment by OlegKapshii** [comment]
@galak Do we need this PR after you have submitted PR with PSoC6 OpenOCD support?

File: boards/arm/cy8ckit_062_wifi_bt_m0/doc/cy8ckit_062_wifi_bt.rst [hunk 1]
```diff
@@ -124,6 +124,40 @@ and J12 that can be used with Segger J-Link.
...
+   $ cd build
+   $ cmake -G"Eclipse CDT4 - Ninja" -DBOARD=cy8ckit_062_wifi_bt_m0 ..
+   $ ninja
+
+Switch the DevKit into CMSIS-DAP mode using SW3 (LED2 should blink) and flash
+the board:
+
+.. code-block:: console
+
+   $<openocd_path>\bin\openocd -c "source [find interface/cmsis-dap.cfg]" -c "transport select swd" -c "source [find target/psoc6.cfg]" -c "if [catch {program {<zephyr_path>\samples\hello_world\build\zephyr\zephyr.elf}} ] { echo {** Program operation failed **} } else { echo {** Program operation completed successfully **} }" -c "reset_config srst_only;reset run;psoc6.dap dpreg 0x04 0x00;shutdown"
```
Comment (dbkinder) [inline boards/arm/cy8ckit_062_wifi_bt_m0/doc/cy8ckit_062_wifi_bt.rst]: Can this be written as this (so the block reads better):

```
$ <openocd_path>\bin\openocd -c "source [find interface/cmsis-dap.cfg]" \
   -c "transport select swd" -c "source [find target/psoc6.cfg]" \
   -c "if [catch {program {<zephyr_path>\samples\hello_world\build\zephyr\zephyr.elf}} ] \
        { echo {** Program operation failed **} } \
        else { echo {** Program operation completed successfully **} }" \
   -c "reset_config srst_only;reset run;psoc6.dap dpreg 0x04 0x00;shutdown"
```

**Review by dbkinder — CHANGES_REQUESTED** [review]

**Comment by galak** [comment]
> @galak Do we need this PR after you have submitted PR with PSoC6 OpenOCD support?

Yeah, if you are going lets close this PR.

**Comment by carlescufi** [comment]
@OlegKapshii @pfalcon note that by default the official way of flashing in Zephyr is `ninja flash`, since that is cross-platform, unlike `make`.

**Comment by galak** [comment]
> Well, I can assure you that it doesn't. For it to work, it should be implemented first. I.e., we want to provided an abstracted and consistent flashing interface for flashing any board. For that, we have "make flash", and you'll see that it now redirects to "west flash". But in the west tool, support for each board needs to be implemented first (hopefully not from scratch, just add needed config I hope - I didn't look into it myself).

I implemented support here:

https://github.com/zephyrproject-rtos/zephyr/pull/11452

**Comment by pfalcon** [comment]
Oh, cool! Can't keep up with all the stuff happening ;-).

**Review by dbkinder — APPROVED** [review]
+1 for doc changes, thanks.

**Review by nashif — APPROVED** [review]

**Pull request merged**